resources/assets/js/components/Application/Intro/IntroPage.tsx

```diff
@@ -44,12 +48,22 @@ export const IntroPage: React.FunctionComponent<IntroPageProps> = ({
           currentTitle={intl.formatMessage(stepNames.welcome)}
           steps={makeProgressBarSteps(
             applicationId,
-            application,
+            steps,
             intl,
             "other",
+            false,
           )}
         />
       )}
+      {showLoadingState && (
+        <h2
+          data-c-heading="h2"
+          data-c-align="center"
+          data-c-padding="top(2) bottom(3)"
+        >
+          {intl.formatMessage(loadingMessages.loading)}
+        </h2>
+      )}
```

Comment:
I don't think we should show this loading state at all, since we're not waiting to show anything.